vision/google/cloud/vision/annotations.py

```diff
@@ -21,6 +21,21 @@
 from google.cloud.vision.safe import SafeSearchAnnotation
 
 
+FACE_ANNOTATIONS = 'faceAnnotations'
+IMAGE_PROPERTIES_ANNOTATION = 'imagePropertiesAnnotation'
+SAFE_SEARCH_ANNOTATION = 'safeSearchAnnotation'
+
+KEY_MAP = {
```

Comment:
Sorry I didn't say so explicitly, but this, and the string constants should be a non-public constant: `_KEY_MAP`